cmd/system-probe/subcommands/runtime/command.go

```diff
@@ -0,0 +1,875 @@
+	policiesDir := evalArgs.dir
+
+	// enabled all the rules
+	enabled := map[eval.EventType]bool{"*": true}
+
+	ruleOpts := rules.NewRuleOpts(enabled)
+	evalOpts := newEvalOpts(evalArgs.windowsModel)
+	ruleOpts.WithLogger(seclog.DefaultLogger)
+
+	agentVersionFilter, err := newAgentVersionFilter()
+	if err != nil {
+		return fmt.Errorf("failed to create agent version filter: %w", err)
+	}
+
+	loaderOpts := rules.PolicyLoaderOpts{
+		MacroFilters: []rules.MacroFilter{
+			agentVersionFilter,
+		},
+		RuleFilters: []rules.RuleFilter{
+			&rules.RuleIDFilter{
+				ID: evalArgs.ruleID,
+			},
+		},
+	}
+
+	provider, err := rules.NewPoliciesDirProvider(policiesDir, false)
+	if err != nil {
+		return err
+	}
+
+	loader := rules.NewPolicyLoader(provider)
+
+	var ruleSet *rules.RuleSet
+	if evalArgs.windowsModel {
+		ruleSet = rules.NewRuleSet(&winmodel.Model{}, newFakeWindowsEvent, ruleOpts, evalOpts)
+	} else {
+		ruleSet = rules.NewRuleSet(&model.Model{}, newFakeEvent, ruleOpts, evalOpts)
+	}
+
+	if err := ruleSet.LoadPolicies(loader, loaderOpts); err.ErrorOrNil() != nil {
+		return err
+	}
+
+	event, err := eventDataFromJSON(evalArgs.eventFile)
+	if err != nil {
+		return err
+	}
+
+	report := EvalReport{
+		Event: event,
+	}
+
+	if !evalArgs.windowsModel {
+		approvers, err := ruleSet.GetApprovers(kfilters.GetCapababilities())
+		if err != nil {
+			report.Error = err
+		} else {
+			report.Approvers = approvers
+		}
+	}
+
+	report.Succeeded = ruleSet.Evaluate(event)
+	output, err := json.MarshalIndent(report, "", "    ")
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%s\n", string(output))
+
+	if !report.Succeeded {
+		os.Exit(-1)
+	}
+
+	return nil
+}
+
+// nolint: deadcode, unused
+func runRuntimeSelfTest(_ log.Component, _ config.Component, _ secrets.Component) error {
+	client, err := secagent.NewRuntimeSecurityClient()
+	if err != nil {
+		return fmt.Errorf("unable to create a runtime security client instance: %w", err)
+	}
+	defer client.Close()
+
+	selfTestResult, err := client.RunSelfTest()
+	if err != nil {
+		return fmt.Errorf("unable to get a process self test: %w", err)
+	}
+
+	if selfTestResult.Ok {
+		fmt.Printf("Runtime self test: OK\n")
+	} else {
+		fmt.Printf("Runtime self test: error: %v\n", selfTestResult.Error)
+	}
+	return nil
+}
+
+func reloadRuntimePolicies(_ log.Component, _ config.Component, _ secrets.Component) error {
+	client, err := secagent.NewRuntimeSecurityClient()
+	if err != nil {
+		return fmt.Errorf("unable to create a runtime security client instance: %w", err)
+	}
+	defer client.Close()
+
+	_, err = client.ReloadPolicies()
+	if err != nil {
+		return fmt.Errorf("unable to reload policies: %w", err)
+	}
+
+	return nil
+}
+
+// StartRuntimeSecurity starts runtime security
+func StartRuntimeSecurity(log log.Component, config config.Component, hostname string, stopper startstop.Stopper, statsdClient ddgostatsd.ClientInterface, wmeta workloadmeta.Component) (*secagent.RuntimeSecurityAgent, error) {
```

Comment:
I don't think it makes sense to port the start command because system-probe already has its dedicated start command.